enclave/src/extrinsic_deposit.rs

```diff
@@ -0,0 +1,47 @@
+use polkadex_sgx_primitives::{AccountId, AssetId};
```

Comment:
Rename the file to something generic, It will be handling all extrinsics signed by enclave

Reply:
done